### PR TITLE
[ISSUE 3407] sub task2 add the logic of annotation on the splicing class for alibaba dubbo client

### PR DESCRIPTION
**Pull request by sunshujie1990**
[ISSUE 3407] sub task2 add the logic of annotation on the splicing class for alibaba dubbo client
https://github.com/apache/incubator-shenyu/issues/3407
Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-examples/shenyu-examples-dubbo/shenyu-examples-alibaba-dubbo-service-annotation/src/main/java/org/apache/shenyu/examples/alibaba/dubbo/service/annotation/impl/DubboMultiParamServiceImpl.java [hunk 1]
```diff
@@ -32,6 +32,7 @@
  * The type Dubbo multi param service.
  */
 @Service
+@ShenyuDubboClient(path = "/demo")
```
[the remark this answers is not in the document]
Comment (dragon-zhang) [reply shenyu-examples/shenyu-examples-dubbo/shenyu-examples-alibaba-dubbo-service-annotation/src/main/java/org/apache/shenyu/examples/alibaba/dubbo/service/annotation/impl/DubboMultiParamServiceImpl.java:35]: And recommended to change only one test class.

**Comment by dragon-zhang** [comment]
@sunshujie1990 Hi, how is your progress now ?

**Comment by sunshujie1990** [comment]
> 

Working on it. I will summit a new commit in the next few days.

**Comment by dragon-zhang** [comment]
please check ci: https://github.com/apache/incubator-shenyu/runs/6775992332?check_suite_focus=true

**Comment by dragon-zhang** [comment]
> > please check ci: https://github.com/apache/incubator-shenyu/runs/6775992332?check_suite_focus=true
> 
> @dragon-zhang Please help me review it again.

sure

**Review by dragon-zhang — APPROVED** [review]

**Comment by dragon-zhang** [comment]
@sunshujie1990 Have fun with contribution ! Please add the wechat Z18179469818

**Pull request merged**